client/src/components/helpers/ToggleButton.js

```diff
@@ -1,74 +1,78 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import {
-  ToggleButtonGroup as BSBG,
-  ToggleButton as TB
+    ToggleButtonGroup as bsbg,
+    ToggleButton as tb
```

Comment:
```suggestion
    ToggleButtonGroup,
    ToggleButton as BootstrapToggleButton
```
How about this?  It's a little verbose, but bsbg and tb don't mean anything unless you look them up or already know what they mean.